src/datatable.js

```diff
@@ -91,6 +91,14 @@ export class DataTable {
       });
   }
 
+  gatherData(criteria = {}) {
+    return this.repository.find(criteria || {}, true).then(result => {
+      return result;
+    }).catch(error => {
+      this.triggerEvent('exception', {on: 'load', error: error});
+    });
```

Comment:
tricky. re.throw the error or not. i see you decided against @RWOverdijk thoughts. i guess when the event is used, re-throwing would just annoy